quinn-proto/src/endpoint.rs

```diff
@@ -1191,8 +1193,7 @@ pub struct Incoming {
     packet: InitialPacket,
     rest: Option<BytesMut>,
     crypto: Keys,
-    retry_src_cid: Option<ConnectionId>,
-    orig_dst_cid: ConnectionId,
+    token_state: IncomingTokenState,
```

Comment:
Nit: suggest ditching both the `_state` and `State` suffixes, I don't think they're helping?